dist/css/components/my-archive.css

```
@@ -0,0 +1,17 @@
+/* Error: Undefined variable.
+ *    ,
+ * 27 | $back-ground: lighten($purple, 20%);
+ *    |                       ^^^^^^^
+ *    '
+ *   src/scss/components/my-archive.scss 27:23  root stylesheet */
+
+body::before {
```

```suggestion
body {
```

This style is currently targetting the :before pseudoselector, which won't affect the rest of the site. This seems like a mistake.

---

Can you explain the purpose of this to me? It looks like an error message from SASS

---

Hi Oliver! Oops, this my my-archive file was just a memo for myself. This is basically a collection of stuff I thought I might use somewhere in future. I forgotten taken it out from the folder actually, definitely not for distribution. I removed the file now. Thanks for spotting.